### PR TITLE
Better highlight which tab is active

### LINKED ISSUE #1623: Highlight which "tab" is selected better in APIView
At a glance, it's not obvious which "tab" is selected when looking at Manual vs. Automatic or Open vs. Closed:

image

In part, this is because there are only 2 per grouping. Is the lighter color selected, or the darker color? It gets even more confusing because for the blue language selection, the lighter color is selected (in this e.g. "All"); however, for the gray buttons it turns out the darker color is selected.

I recommend either being consistent with whether lighter or darker means "selected", or just make it even more obvious in any case and add a ✔ or something to the button to make it obvious which is selected.

### PR DESCRIPTION
**Pull request by heaths**
Fixes #1623

### CODE REVIEW

**Comment by heaths** [comment]
Wasn't really able to test the ASP.NET change, but it's pretty straight forward to fix the expressions for the active tab. The CSS I tested in F12 tools in Edge.

@praveenkuttappan could you update the internal wiki (or somewhere) which dev test settings?

**Comment by praveenkuttappan** [comment]
I have queued a job to deploy this to staging instance. We can get this merged after it is deployed to staging. We use staging instance for testing for the time being. But I can create a new dev test resources and update wiki with those details.

File: src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Index.cshtml [hunk 1]
```diff
@@ -16,14 +16,14 @@
...
-            <a class="btn btn-primary" active-if='@Model.Language != "Python"' asp-all-route-data=@Model.GetRoutingData(language: "Python")>Python</a>
+            <a class="btn btn-primary" active-if='@Model.Language == "All"' asp-all-route-data=@Model.GetRoutingData(language: "All")>All</a>
+            <a class="btn btn-primary" active-if='@Model.Language == "C"' asp-all-route-data=@Model.GetRoutingData(language: "C")>C</a>
+            <a class="btn btn-primary" active-if='@Model.Language == "C#"' asp-all-route-data=@Model.GetRoutingData(language: "C#")>C#</a>
+            <a class="btn btn-primary" active-if='@Model.Language == "C++"' asp-all-route-data=@Model.GetRoutingData(language: "C++")>C++</a>
+            <a class="btn btn-primary" active-if='@Model.Language == "Go"' asp-all-route-data=@Model.GetRoutingData(language: "Go")>Go</a>
+            <a class="btn btn-primary" active-if='@Model.Language == "Java"' asp-all-route-data=@Model.GetRoutingData(language: "Java")>Java</a>
+            <a class="btn btn-primary" active-if='@Model.Language == "JS/TS"' asp-all-route-data=@Model.GetRoutingData(language: "JavaScript")>JS/TS</a>
```
Comment (pakrym) [inline src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Index.cshtml:25]:
```suggestion
            <a class="btn btn-primary" active-if='@Model.Language == "JavaScript"' asp-all-route-data=@Model.GetRoutingData(language: "JavaScript")>JS/TS</a>
```

File: src/dotnet/APIView/APIViewWeb/Client/css/site.scss [hunk 2]
```diff
@@ -446,6 +446,21 @@ code-inner {
     cursor: pointer;
 }
 
+.btn-group {
+    .btn.active {
+        border-radius: .25rem !important;
+    }
+
+    .btn:not(.active) {
+        transform: scaleY(0.9);
```
Comment (pakrym) [inline src/dotnet/APIView/APIViewWeb/Client/css/site.scss:455]: nit: downscaling messes with fonts a bit, consider upscaling the active one.

**Review by pakrym — APPROVED** [review]

File: src/dotnet/APIView/APIViewWeb/Client/css/site.scss (hunk 2 above)
Comment (heaths) [reply src/dotnet/APIView/APIViewWeb/Client/css/site.scss:455]: Scaling either way will. I did originally consider upscaling the active one, but the buttons are already pretty big. But at a 10% scale either direction, it's not that bad. Originally I was trying to merely make the top a little higher and the text aligned, but there's so many styles (mostly from bootstrap) here that it was more difficult for an internal tool than I wanted to spend time on. Someone better at CSS3 (I stopped doing most web dev circa CSS2 debut) is welcome to improve on this. :)

File: src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Index.cshtml (hunk 1 above)
Comment (heaths) [reply src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Index.cshtml:25]: Good catch!

**Comment by JoshLove-msft** [comment]
> I have queued a job to deploy this to staging instance. We can get this merged after it is deployed to staging. We use staging instance for testing for the time being. But I can create a new dev test resources and update wiki with those details.

+1 for updating the wiki with the new resource information. I noticed that localHost testing no longer works with the local settings I was using.

**Comment by heaths** [comment]
Same here, @JoshLove-msft . @praveenkuttappan is going to look into that. I get an error when GitHub is calling back after authentication.

**Comment by JoshLove-msft** [comment]
> Same here, @JoshLove-msft . @praveenkuttappan is going to look into that. I get an error when GitHub is calling back after authentication.

Yep same error

**Pull request merged**